contracts/core/Allo.sol

```diff
@@ -716,7 +716,7 @@ contract Allo is
     }
 
     /// @dev Logic copied from ERC2771ContextUpgradeable OZ contracts
-    function _msgData() internal view virtual override returns (bytes calldata) {
```

Comment:
I had to remove the `virtual` keyword so smock does not crash when generating the mock. Looks like the return paramater of type `calldata` is messing with the generation.